utils/fetchLsp8Metadata.ts

```diff
@@ -151,7 +151,9 @@ const getLsp8Metadata = async (
 
     // in order to get full url we combine URI it with tokenId (must be lowercased)
     // we also resolve url as uri might be ipfs link
-    url = resolveUrl(uri + tokenIdParsed(tokenId, tokenIdFormat).toLowerCase())
+    url = resolveUrl(
+      uri.trim() + tokenIdParsed(tokenId, tokenIdFormat).toLowerCase()
```

Comment:
Although it says toLowerCase in the docs it just doesn't make sense so I think tokenIdParsed should be doing the lowercasing so that it can be adjusted easily if it's required.

Reply:
Alright, moving lowercase to `tokenIdParsed` then.